rllib/tuned_examples/a2c/cartpole_a2c.py

```diff
@@ -0,0 +1,11 @@
+from ray.rllib.algorithms.a2c import A2CConfig
+
+
+config = (
+    A2CConfig()
+    .training(lr=0.001)
+    .framework("tf")
+    .rollouts(num_rollout_workers=0)
+    .environment(env="CartPole-v1")
```

Comment:
Up to you, but I feel like doing .environment("CartPole-v1") looks nicer (without the env=).